### ISSUE TITLE
need option to ignore infinite loop entries, and just delete the empty folders

### ISSUE THREAD

**Issue by therobyouknow** [hxseven/Remove-Empty-Directories#17]
on v2.2, using Windows 10 Pro 64bit.

It finds empty folders but also finds 'infinite loop detected' - what actually is this? when I open the folder where this is found, I can't see a symlink, so what else could it be?

At the end of the scan, I have a number of 'infinite loop detected', but the delete empty folders button is greyed out - why?  Is it possible to tell the problem to exclude the infinite loop detected entries and just delete the empty folders.  I looked in settings, and could not see an option for this.

tried v2.3 beta, but this seemed to hang, or didn't give clear indication on progress.

**Comment by Masamune3210** [comment]
Infinite loop entries can be caused by symlinks, hardlinks, or filesystem corruption usually

**Comment by therobyouknow** [comment]
> Infinite loop entries can be caused by symlinks, hardlinks, or filesystem corruption usually

Right, and I'm already aware. So, sorry that doesn't answer my request. So can the program ignore them then? And just delete the empty folders it has found?

After all, it provides a list of empty folders, so can it just go ahead and delete them?

**Comment by therobyouknow** [comment]
> Infinite loop entries can be caused by symlinks, hardlinks, or filesystem corruption usually

And also - it would be helpful to give more detail about why it thinks something is an infinite loop - as in which of the above things you mention it thinks that infinite loop it is.  Because, as I had mentioned, when I went to look I could not see why.

Once again, it should still be able to proceed to delete the empty folders it has found.

Thank you for your input.

**Comment by JustMyGithub** [comment]
@therobyouknow see also https://github.com/hxseven/Remove-Empty-Directories/issues/8
